doc/stdlib/base.rst

```diff
@@ -165,166 +227,164 @@ All Objects
 
-   While it isn't normally necessary, user-defined types can override the default ``deepcopy`` behavior by defining a specialized version of the function ``deepcopy_internal(x::T, dict::ObjectIdDict)`` (which shouldn't otherwise be used), where ``T`` is the type to be specialized for, and ``dict`` keeps track of objects copied so far within the recursion. Within the definition, ``deepcopy_internal`` should be used in place of ``deepcopy``, and the ``dict`` variable should be updated as appropriate before returning.
+   While it isn't normally necessary, user-defined types can override the default "deepcopy" behavior by defining a specialized version of the function "deepcopy_internal(x::T, dict::ObjectIdDict)" (which shouldn't otherwise be used), where "T" is the type to be specialized for, and "dict" keeps track of objects copied so far within the recursion. Within the definition, "deepcopy_internal" should be used in place of "deepcopy", and the "dict" variable should be updated as appropriate before returning.
 
-.. function:: isdefined([object,] index | symbol)
 
-   Tests whether an assignable location is defined. The arguments can be an
-   array and index, a composite object and field name (as a symbol), or a
-   module and a symbol.
-   With a single symbol argument, tests whether a global variable with that
-   name is defined in ``current_module()``.
+.. function:: isdefined([object], index | symbol)
+
+   Tests whether an assignable location is defined. The arguments can be an array and index, a composite object and field name (as a symbol), or a module and a symbol. With a single symbol argument, tests whether a global variable with that name is defined in "current_module()".
+
 
 .. function:: convert(T, x)
 
-   Convert ``x`` to a value of type ``T``.
+   Convert "x" to a value of type "T".
 
-   If ``T`` is an ``Integer`` type, an :exc:`InexactError` will be raised if
-   ``x`` is not representable by ``T``, for example if ``x`` is not
-   integer-valued, or is outside the range supported by ``T``.
+   If "T" is an "Integer" type, an "InexactError" will be raised if "x" is not representable by "T", for example if "x" is not integer-valued, or is outside the range supported by "T".
 
-   .. doctest::
+   ::
 
-      julia> convert(Int, 3.0)
-      3
+       julia> convert(Int, 3.0)
+       3
 
-      julia> convert(Int, 3.5)
-      ERROR: InexactError()
-       in convert at int.jl:196
+       julia> convert(Int, 3.5)
+       ERROR: InexactError()
+        in convert at int.jl:196
 
-   If ``T`` is a :obj:`FloatingPoint` or :obj:`Rational` type, then it will return
-   the closest value to ``x`` representable by ``T``.
+   If "T" is a "FloatingPoint" or "Rational" type, then it will return the closest value to "x" representable by "T".
 
-   .. doctest::
+   ::
```

Comment:
In my experiments I was converting this to

    ```doctest
    code to test
    ```

That way it could be extracted. I don't recall if that;s the way it's done in Docile.

Kind of annoying all this meta data is just being tossed away.